### PR TITLE
update python packages

### PR DESCRIPTION
**Pull request by naveen521kk**
- mingw-w64-python-colorama
- mingw-w64-python-ipython
- mingw-w64-python-jedi
- mingw-w64-python-prompt_toolkit
- mingw-w64-python-ptyprocess
- mingw-w64-python-pygments
- mingw-w64-python-sympy
- mingw-w64-python-traitlets
- mingw-w64-python-virtualenv

Dependencies are checked automatically...

### CODE REVIEW

File: mingw-w64-python-ipython/PKGBUILD [hunk 1]
```diff
@@ -6,27 +6,25 @@ pkgname=("${MINGW_PACKAGE_PREFIX}-python-${_realname}")
...
+         "${MINGW_PACKAGE_PREFIX}-python-prompt_toolkit"
          "${MINGW_PACKAGE_PREFIX}-python-pygments"
-         "${MINGW_PACKAGE_PREFIX}-python-backcall"
-         "${MINGW_PACKAGE_PREFIX}-python-pexpect"
-         "${MINGW_PACKAGE_PREFIX}-python-colorama"
-         "${MINGW_PACKAGE_PREFIX}-python-win_unicode_console")
+         "${MINGW_PACKAGE_PREFIX}-python-setuptools"
+         "${MINGW_PACKAGE_PREFIX}-python-traitlets"
+         "${MINGW_PACKAGE_PREFIX}-sqlite3"
+         "winpty")
```
Comment (lazka) [inline mingw-w64-python-ipython/PKGBUILD:27]: why winpty?

Comment (naveen521kk) [reply mingw-w64-python-ipython/PKGBUILD:27]: don't know it was previously there at top.

Comment (lazka) [reply mingw-w64-python-ipython/PKGBUILD:27]: ah I missed that

Comment (naveen521kk) [reply mingw-w64-python-ipython/PKGBUILD:27]: that was just some sorting done :wink:

**Pull request merged**